tokio/src/macros/support.rs

```diff
@@ -4,6 +4,10 @@ cfg_macros! {
     pub use crate::util::thread_rng_n;
 }
 
+// Re-export not needed since it's pulling from std
+#[deprecated(note = "Prefer importing std::future::Future directly")]
 pub use std::future::Future;
+#[deprecated(note = "Prefer importing std::pin::Pin directly")]
 pub use std::pin::Pin;
+#[deprecated(note = "Prefer importing std::task::Poll directly")]
 pub use std::task::Poll;
```

Comment:
Note that this module is not a public API

https://github.com/tokio-rs/tokio/blob/9a3ce91ef54210d418318696dccdf0d14fe2b5fd/tokio/src/lib.rs#L365-L371